### ISSUE TITLE
Changing loss function

### ISSUE THREAD

**Issue by mortentang** [Megvii-BaseDetection/YOLOX#1080]
I would like to change the BCEWithLogitsLoss to CrossEntropyLoss, as I am working on a dataset with single-labels.
However, I have been unable to do so.

Would I need to change the format a lot for the classification part, or is this easy to implement?

**Comment by FateScript** [comment]
I believe that it's easy to implement.
